
this software has an internal hostbuild of a subproject for upstream cross-compiling
support, but that hostbuild fails no matter whether on-device building or
cross-compiling is used, so it must be disabled and the non-cross-compilation
codepath forced

--- a/src/indexer/CMakeLists.txt
+++ b/src/indexer/CMakeLists.txt
@@ -3,7 +3,7 @@
 if(CMAKE_CROSSCOMPILING AND KATEHIGHLIGHTINGINDEXER_EXECUTABLE)
     add_executable(katehighlightingindexer IMPORTED GLOBAL)
     set_target_properties(katehighlightingindexer PROPERTIES IMPORTED_LOCATION ${KATEHIGHLIGHTINGINDEXER_EXECUTABLE})
-elseif(CMAKE_CROSSCOMPILING)
+elseif(FALSE)
     include(ECMQueryQt)
     ecm_query_qt(NATIVE_PREFIX QT_HOST_PREFIX)
     message(STATUS "Building katehighlightingindexer against ${NATIVE_PREFIX}")
